push_physical_badging_records.ps1

```diff
@@ -62,7 +62,7 @@ function RetryCommand {
         [scriptblock]$ScriptBlock,
 
         [Parameter(Position = 1, Mandatory = $false)]
-        [int]$Maximum = 15
+        [int]$Maximum = 5
```

Comment:
How much is the duration between each retry?

Reply:
60 seconds by default, it is present as a parameter "retryTimeout" when script is invoked from powershell